azure-pipelines-deploy-template.yml

```diff
@@ -346,7 +346,32 @@ jobs:
               SpecifySlotOrASE: true
               ResourceGroupName: '$(resourceGroupName)'
               Slot: staging
+          
+          - task: AzurePowerShell@4
+            displayName: 'GET /check in $(appServiceName)-staging'
+            inputs:
+              azureSubscription: '${{parameters.subscriptionName}}'
+              ScriptType: InlineScript
+              azurePowerShellVersion: LatestVersion
+              Inline: |
+                Param(
+                  [string] $appServiceName = "$(appServiceName)"
+                )
+                $healthCheckUrl = [string]::Format(“https://{0}-staging.azurewebsites.net/check”, $appServiceName)
+
+                try
+                {
+                    $response = Invoke-WebRequest -Uri $healthCheckUrl -TimeoutSec 120 -ErrorAction Stop
+                    $statusCode = $response.StatusCode
+                }
+                catch
+                {
+                    $statusCode = $_.Exception.Response.StatusCode.value__
+                }
 
+                if($statusCode -ne '200'){
+                  Write-Error "$healthCheckUrl returned status $statusCode"
```

Comment:
My PowerShell Azure-Fu is weak, does `Write-Error` stop the build from going further?

Reply:
yes `Write-Error` fails a task, is also used elsewhere in the scripts. 👍